File: backend/src/models/TDONHANGHUYCHITIET.ts
import { Model, DataTypes } from 'sequelize';
import db from '../config/database';

class TDONHANGHUYCHITIET extends Model {
    public id!: any;
    public note!: any;
    public tdonhanghuyid!: any;
    public status!: any;
    public usermodifiedid!: any;
    public timemodified!: any;
    public timecreated!: any;
    public usercreatedid!: any;
    public mahang!: any;
    public tenhang!: any;
    public dvt!: any;
    public dongia!: any;
    public thanhtien!: any;
    public soluong!: any;
}

TDONHANGHUYCHITIET.init({
    ID: {
        type: DataTypes.INTEGER,
        primaryKey: true, autoIncrement: true,
    },
    NOTE: {
        type: DataTypes.STRING,
        
    },
    TDONHANGHUYID: {
        type: DataTypes.INTEGER,
        
    },
    STATUS: {
        type: DataTypes.BOOLEAN,
        
    },
    USERMODIFIEDID: {
        type: DataTypes.INTEGER,
        
    },
    TIMEMODIFIED: {
        type: DataTypes.DATE,
        
    },
    TIMECREATED: {
        type: DataTypes.DATE,
        
    },
    USERCREATEDID: {
        type: DataTypes.INTEGER,
        
    },
    MAHANG: {
        type: DataTypes.STRING,
        
    },
    TENHANG: {
        type: DataTypes.STRING,
        
    },
    DVT: {
        type: DataTypes.STRING,
        
    },
    DONGIA: {
        type: DataTypes.DECIMAL(18,2),
        
    },
    THANHTIEN: {
        type: DataTypes.DECIMAL(18,2),
        
    },
    SOLUONG: {
        type: DataTypes.DECIMAL(18,2),
        
    },
}, {
    sequelize: db.sequelize,
    modelName: 'TDONHANGHUYCHITIET',
    tableName: 'TDONHANGHUYCHITIET',
    timestamps: false
});

export default TDONHANGHUYCHITIET;

File: frontend/Models/TDONHANGHUYCHITIET.cs
using System;

namespace QuanLyBar.Client.Models
{
    public class TDONHANGHUYCHITIET
    {
        public int? Id { get; set; }
        public string Note { get; set; }
        public int? TdonhanghuyId { get; set; }
        public bool? Status { get; set; }
        public int? UsermodifiedId { get; set; }
        public DateTime? Timemodified { get; set; }
        public DateTime? Timecreated { get; set; }
        public int? UsercreatedId { get; set; }
        public string Mahang { get; set; }
        public string Tenhang { get; set; }
        public string Dvt { get; set; }
        public decimal? Dongia { get; set; }
        public decimal? Thanhtien { get; set; }
        public decimal? Soluong { get; set; }
    }
}
